browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -3173,6 +3176,9 @@ If you don't accept this request, VPN will not reconnect and your internet conne
     <message name="IDS_WALLET_SOL_NAME" desc="Name of SOL cointype">
       Solana
     </message>
+    <message name="IDS_WALLET_FIL_NAME" desc="Name of FIL cointype">
```

Comment:
perhaps we should mark as `translateable="false"` here as well as `Solana` above.